### ISSUE TITLE
The Ortho loss in the paper Eq.10

### ISSUE THREAD

**Issue by HelenMao** [willylulu/RelGAN#2]
Hi, in your paper you mentioned the ortho loss in Eq.10.
However, I did not find the corresponding codes in your Keras version implementation.
In the Pytorch version, there are the corresponding codes.
So in your final implementation, did you include that loss or not?
Thanks!

**Comment by willylulu** [comment]
It doesn't like pytorch you need write regular loss in your total loss. 
In keras, we write the regularizer in each convolution layer
For example:
https://github.com/willylulu/RelGAN/blob/5d9faf87240554c1b2565b610598fac7f1061557/module.py#L39

you can see the function input kernel_regularizer = orthogonal
and we implement this function orthogonal in ops.py

**Comment by HelenMao** [comment]
Got it, thanks!
Also, I find you use lecun_norm for kernel initializer in several convolutions of the discriminator in Keras version.
It will have much impact using such weighting initialization?

**Comment by willylulu** [comment]
we set he normal just because progressive gan use he normal
but actually it's not important. we can't see any different

**Comment by HelenMao** [comment]
Ok, thanks!
